inst/include/RcppArmadilloAs.h

```diff
@@ -125,7 +127,7 @@ namespace traits {
         typedef typename Rcpp::traits::storage_type<RTYPE>::type value_t;
         Exporter(SEXP x) : vec(x) {}
       
-        cube_t get() {
+        cube_t get(bool copy = true) {
```

Comment:
That was the old behaviour, then we never get copies (see below). Only way I could circumvent this was to use copy as default here, but perhaps there is a more clever way.